src/modm/architecture/interface/accessor_flash.hpp.in

```diff
@@ -186,6 +186,9 @@ asFlash(const T* ptr)
 }	// namespace modm
 
 %% if with_io
+namespace modm
+{
+
 template<typename T>
 modm::IOStream&
 operator << (modm::IOStream& os, modm::accessor::Flash<T> ptr);
```

Comment:
I could not find the definition of it. To me it looks like it's declared but not defined anywhere.

Reply:
The one below is declared static but I don't see why. It says it is a "specialization for `char`" (but actually isn't a template specialization and this is a good thing). Maybe that's why someone declared the version above.
```
/**
 * Streamoperator - specialization for `char`
 * @ingroup modm_architecture_accessor
 */
static inline modm::IOStream&
operator << (modm::IOStream& os, modm::accessor::Flash<char> ptr)
```

Reply:
Agreed, the above template and the `static` should be removed.